### PR TITLE
Add release flag to prior-1.32 blog posts

### PR DESCRIPTION
**Pull request by Folyd**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ site
 static/styles/vendor.css
 static/styles/app.css
 static/styles/fonts.css
+.idea
```
Comment (Mark-Simulacrum) [inline .gitignore:8]: We generally prefer not to include editor-specific files in the project .gitignore (I see .bk and .DS_Store here which seem like they could get removed by a separate PR, FWIW) -- can you drop this change?

File: src/main.rs [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 use chrono::Timelike;
 use handlebars::{handlebars_helper, Handlebars};
 use sass_rs::{compile_file, Options};
 use serde_derive::Serialize;
 use serde_json::json;
-use std::convert::AsRef;
-use std::error::Error;
-use std::fs::{self, File};
-use std::io::Write;
-use std::path::{Path, PathBuf};
```
Comment (Mark-Simulacrum) [inline src/main.rs:15]: Can we avoid whatever reformatting or shuffling was done here? It's going to make it harder to review and accept this PR -- and it shouldn't be necessary?

**Review by Mark-Simulacrum — COMMENTED** [review]
Can you say more about your intent with this PR? It looks like it's aiming to add the version number for releases, but currently Rust infra doesn't consume that. I am not see any optimizations either, so not quite sure what you meant there.

I would prefer to drop the clippy and .gitignore changes from this PR regardless.

File: .gitignore (hunk 1 above)
Comment (bjorn3) [reply .gitignore:8]: If you don't know it yet, you can have `$XDG_CONFIG_HOME/git/ignore` for global ignores and `.git/info/exclude` for not committed per-repo ignores.

**Comment by Folyd** [comment]
> Can you say more about your intent with this PR? It looks like it's aiming to add the version number for releases, but currently Rust infra doesn't consume that. I am not see any optimizations either, so not quite sure what you meant there.
> 
> I would prefer to drop the clippy and .gitignore changes from this PR regardless.

Thanks for reviewing. I rectified the title of this PR and dropt the clippy and .gitignore changes.

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**